user/articles/3-layout-and-design/12-widget-reference.markdown

```diff
@@ -1,41 +1,30 @@
 # Commerce Widgets
 
-Liferay Commerce ships with a wide range of widgets you'll use to construct
-your site. This section provides a convenient overview in one place.
+Liferay Commerce ships with a wide range of widgets you'll use to construct your
+site. This section provides a convenient overview in one place.
 
 ## Account Management
```

Comment:
The revised version is, more or less, the appropriate length for a description in my humble view.